examples/advanced-project/src/__checks__/multi-step-spacex.check.ts

```diff
@@ -0,0 +1,21 @@
+import * as path from 'path'
+import { MultiStepCheck } from 'checkly/constructs'
+import { smsChannel, emailChannel } from '../alert-channels'
+
+const alertChannels = [smsChannel, emailChannel]
+
+/*
+* In this example, we bundle all basic checks needed to check the Checkly homepage. We explicitly define the Browser
+* check here, instead of using a default based on a .spec.js file. This allows us to override the check configuration.
+* We can also add more checks into one file, in this case to cover a specific API call needed to hydrate the homepage.
+*/
+
+// We can define multiple checks in a single *.check.ts file.
+new MultiStepCheck('spacex-multistep-check', {
```

Comment:
Maybe we should mention what exactly a multi-step API check is and how it works?

Also, this available for free plans? If not, maybe just a brief comment explaining?

Reply:
Afaik the initial decision on pricing is that it will be available on all plan types and users will be able to have unlimited multi-step checks, like with API/Browser.